types/lib/operators.d.ts

```diff
@@ -481,4 +481,4 @@ declare const Op: {
   readonly values: unique symbol;
 };
 
-export = Op;
+export = Op;
```

Comment:
This change is not wanted

Reply:
look this changed the line backj to LF. which is follow our eslint rules.

Reply:
GitHub shows me that there is no empty line at the end of the file, which is preferred